server/protocols/protocols-smtp/src/main/java/org/apache/james/smtpserver/AddDefaultAttributesMessageHook.java

```diff
@@ -53,11 +56,11 @@ public HookResult onMessage(SMTPSession session, Mail mail) {
             mailImpl.setRemoteHost(session.getRemoteAddress().getHostName());
             mailImpl.setRemoteAddr(session.getRemoteAddress().getAddress().getHostAddress());
             if (session.getUser() != null) {
-                mail.setAttribute(Mail.SMTP_AUTH_USER_ATTRIBUTE_NAME, session.getUser());
+                mail.setAttribute(new Attribute(Mail.SMTP_AUTH_USER_ATTRIBUTE_NAME, AttributeValue.of(session.getUser())));
             }
 
             if (session.isRelayingAllowed()) {
-                mail.setAttribute(SMTP_AUTH_NETWORK_NAME, "true");
+                mail.setAttribute(new Attribute(SMTP_AUTH_NETWORK_NAME, AttributeValue.of("true")));
```

Comment:
Why not a `Boolean`?

Reply:
I hesitated to do this remark, but we are not here to change meaning of the mailets.